apps/datahub/src/app/header/header-record/header-record.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component, ChangeDetectionStrategy } from '@angular/core'
+import { MdViewFacade } from '@geonetwork-ui/feature/record'
+import { RouterFacade } from '@geonetwork-ui/feature/router'
+
+@Component({
+  selector: 'datahub-header-record',
+  templateUrl: './header-record.component.html',
+  styleUrls: ['./header-record.component.css'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class HeaderRecordComponent {
+  constructor(
+    public mdViewFacade: MdViewFacade,
+    private searchRouter: RouterFacade
+  ) {}
+  back() {
+    this.searchRouter.back()
```

Comment:
This is a big **no**: using a back action is the best way to have inconsistent navigation. E.g. if I open a datahub record without going through the search, and I press this button, I will actually be taken back to my previous location which could be anything!

Here we need to take the user back to search instead

Reply:
was changed in the other PR yes

Reply:
saw it, thanks! :+1: